File: program/find-cube-root-of-a-number/FindCubeRootOfANumber.c
#include <stdio.h>
#include <math.h>

int main()
{
    double num;

    printf("Enter a number: ");
    scanf("%lf", &num); 

    double result = cbrt(num); 

    printf("Cube root of %.2f is    :    %.2f\n", num, result);
}

File: program/find-cube-root-of-a-number/FindCubeRootOfANumber.cs
using System;
class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter a number : ");
    double num = Convert.ToDouble(Console.ReadLine());
    double cubeRoot = Math.Cbrt(num);
    Console.WriteLine("Cube root of given number : "+cubeRoot);
  }
}
